ansible/roles/common/tasks/main.yml

```diff
@@ -92,6 +96,68 @@
     tags:
       - upgrade
 
+  - name: Get os_upgrades stats
+    shell:
+      cmd: |
+        sed -n "/$(date '+%Y-%m')/,+2p" updates.log | # Groups of lines from the current month
+        grep 'packages' | # Only lines matching 'packages'
+        grep -P -o '(^\d+)' | #Extract the numbers at the beginning of the lines
+        awk '{s+=$1} END {print s}' # Sum all the lines
+    args:
+      chdir: /opt/subspace
+    register: stats_os_upgrades
+    when: send_stats == true and stats_url is defined and stats_api_key is defined
+    tags:
+      - stats
```

Comment:
Can this also have `upgrade` or a new `maintenance` tag or something? I want to be able to just run one thing if possible.

Reply:
Not sure exactly, I usually just comment out everything but
- common
- ruby-common
- rails
roles then provision. We probably want more/less that just that. I think basically what i want is like a "do everything that doesn't need to interfere with the web server." Then we can just run that once a month for maintenance and stat collection.